src/dotnet-roslyn-tools/Commands/PRTaggerCommand.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the.NET Foundation under one or more agreements.
+// The.NET Foundation licenses this file to you under the MIT license.
+// See the License.txt file in the project root for more information.
+
+namespace Microsoft.RoslynTools.Commands;
+
+using System.CommandLine.Invocation;
+using System.CommandLine;
+using Microsoft.RoslynTools.Authentication;
+
+internal static class PRTaggerCommand
+{
+    private static readonly PRTaggerCommandDefaultHandler s_prTaggerCommandHandler = new();
+
+    private static readonly Option<string> ProductName = new(new[] { "--product-name", "-p" }, "Name of product (e.g. 'roslyn' or 'razor')") { IsRequired = true };
```

Comment:
We can use `.FromAmong(string[])` to ensure a valid product name. See https://docs.microsoft.com/en-us/dotnet/standard/commandline/define-commands#list-valid-argument-values